src/server/auth/auth.router.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+
+export default class AuthRouter {
+  constructor(auth, database) {
+    this.router = express.Router();
+    this.auth = auth;
+    this.database = database;
+
+    const validateAuthCallback = (req, res) => {
+      if (req.user) {
+        res.json({
+          success: true,
+          message: 'User is currently authenticated',
+          user: req.user,
+          cookies: req.cookies
```

Comment:
Can `req.cookies` be null?

Reply:
Only if there are other fields missing as well